client/layout/style.scss

```diff
@@ -82,6 +82,18 @@
 			margin-top: 0 !important;
 		}
 	}
+
+	&.has-woocommerce-navigation {
+		#wpbody {
```

Comment:
Just want to note that this may need to be updated depending on the follow up to https://github.com/woocommerce/woocommerce-admin/pull/6337#discussion_r576431976 cc @joelclimbsthings

Reply:
Right, going to get to this shortly, but my inclination is that we'll be changing everything to target the `body` as @joshuatf suggested.

Reply:
Doing the refactor described in https://github.com/woocommerce/woocommerce-admin/pull/6337#discussion_r576431976 seems low priority compared to remaining issues slated for 2.1. Would it be better to include this change, then work on the refactor during Cooldown?

Reply:
I don't mind merging this one in first and @joelclimbsthings can update in his PR, but I'd prefer to not start layering on a bunch of CSS hacks if we can avoid it.

That led us to the untangling a web of bugs that spanned across the app.